### DISCUSSION TITLE
MacBook Cannot sleep while AlDente(free) is discharging

### DISCUSSION THREAD

**Discussion by GvXV3toYUNbmP** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#512] [Q&A] [answered]
Hello
my m1max machine cannot sleep while latest version of Aldente (free) is discharging
Basically sleep menu is disabled and I can only make sleep work if I stop discharging it by Aldente
currently 'Disable Sleep Until Charge Limit' option is disabled as I am free version user.

- Is it known issue on some machine or on latest version OSX/AlDente ?
- If I purchase pro version, will that issue go away by enable/disable it ?

Thanks

I tested old version and sleep on my MacBook is working fine 1.08

I have M1 max MacBook and using latest OSX version.
Also was using latest version of AlDente for the issue

**Comment by bluescorpion-ai** [comment]
11 days and  no one answerer. and they want to sell pro version. yeah sure m8s

**Reply by DamonHD** [reply]
Why be so rude and nasty?  This sort of behaviour drives people away from retail and user-facing roles.

This may or may not be their main day job, and in any case most of Europe is only just emerging from holidays (and today may be another one in some places).

**Comment by MatthiasKerbl** [comment, marked as answer]
> Hello my m1max machine cannot sleep while latest version of Aldente (free) is discharging Basically sleep menu is disabled and I can only make sleep work if I stop discharging it by Aldente currently 'Disable Sleep Until Charge Limit' option is disabled as I am free version user.
> 
> * Is it known issue on some machine or on latest version OSX/AlDente ?
> * If I purchase pro version, will that issue go away by enable/disable it ?
> 
> Thanks
> 
> I tested old version and sleep on my MacBook is working fine 1.08
> 
> I have M1 max MacBook and using latest OSX version. Also was using latest version of AlDente for the issue

Hi @GvXV3toYUNbmP,

I explained the current situation about discharging in the free version here: #495 

Cheers,
Matthias
